organizations/user.go

```diff
@@ -611,3 +612,57 @@ func (oh *OrganizationHandler) ReactivateMember(w http.ResponseWriter, r *http.R
 	}
 	utils.GetSuccess("successfully reactivated member", nil, w)
 }
+
+func InviteMembers(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+
+	// user_collection := "users"
+
+	// sOrgId := mux.Vars(r)["id"]
+	// orgId, err := primitive.ObjectIDFromHex(sOrgId)
+
+	// if err != nil {
+	// 	utils.GetError(errors.New("invalid id"), http.StatusBadRequest, w)
+	// 	return
+	// }
+
+	requestData := make(map[string][]string)
+
+	if err := utils.ParseJsonFromRequest(r, &requestData); err != nil {
+		utils.GetError(err, http.StatusUnprocessableEntity, w)
+		return
+	}
+
+	to := requestData["receivers"]
+
+	configs := utils.NewConfigurations()
+
+	// Sender data.
+	from := configs.SmtpUsername
+	password := configs.SmtpPassword
+
+	// Receiver email address.
+	// to := []string{
+	// 	"wakyutuk@gmail.com",
+	// 	"utukphd@gmail.com",
+	// }
+
+	// smtp server configuration.
+	smtpHost := "smtp.gmail.com"
+	smtpPort := "587"
+
+	// Message.
+	message := []byte("Workspace invitation!")
+
+	// Authentication.
+	auth := smtp.PlainAuth("", from, password, smtpHost)
+
+	// Sending email.
+
```

Comment:
Why recreating this since we already have a function to send mail, mail will not be sent in production because production is using mailgun. your config is to enable you to send on local.